docs/usage.rst

```diff
@@ -445,7 +445,11 @@ To compare two versions depends on your type:
 
 Other types cannot be compared.
 
-If you need to convert some types into other, refer to :ref:`sec.convert.versions`.
+If you need to convert some types into others, refer to :ref:`sec.convert.versions`.
+
+The use of these comparison operators also implies that you can also use builtin
+functions that leverage this capability; builtins including but not limited to: :func:`max`, :func:`min`
+(refer to :ref:`sec_max_min` for examples) and :func:`sorted`.
```

Comment:
From what I remember in my French classes, semicolons are used just like a period (to end a sentence), but highlight the link between the two different sentences. I believe it to be relatively similar in English.
I agree that the comma should be there, this sentence lacked pacing
```suggestion
functions that leverage this capability; builtins including, but not limited to, : :func:`max`, :func:`min`
(for examples, see :ref:`sec_max_min`) and :func:`sorted`.
```

Reply:
I'm not a native speaker so I can only cite what my English proofeader told me. :wink:  From what I've heard, the use of semicolons *looks* simple, but it actually isn't. But I'm not an expert, so that's why I wrote that I'm not sure.

I don't want to force that upon you, it was just meant as an idea. You can only apply the commas and leave the other as is, if you like. Or rewrite the sentence. :wink: